ene-jun-2018/Juan Carlos Sleiman/Practica 1/hola.go

```diff
@@ -0,0 +1,26 @@
+ package main
+
+import (
+"bufio"
+"fmt"
+"os"
+"time"
+)
+
+var text string
+
+func say(s string) {
+for i := 0; i < 10; i++ {
+time.Sleep(100 * time.Millisecond)
+fmt.Println(s)
+fmt.Println(text)
+}
+}
+
+func main() {
+reader := bufio.NewReader(os.Stdin)
```

Comment:
Cuando tratas de hacerlo con bufio al parecer solo una gorutine la puede usar al mismo tiempo: mira [esto](https://stackoverflow.com/questions/33766251/golang-os-stdin-as-a-reader-in-goroutines) es más sencillo si usas `fmt.Scan()`